src/components/ReviewCard/index.tsx

```diff
@@ -2,34 +2,31 @@ import { Fragment } from 'react';
 
 import classNames from 'classnames/bind';
 
-import { getFormatDate } from '@/utils';
-
-import Avatar from '@/components/commons/Avatar';
-import StarRating from '@/components/commons/StarRating';
-
-import { Reviewer } from '@/types';
+import Avatar from '@/components/Avatar';
+import StarRating from '@/components/StarRating';
 
 import styles from './ReviewCard.module.scss';
 
 const cx = classNames.bind(styles);
 
 type ReviewCardProps = {
-  user: Reviewer;
+  profileImageUrl: string | null;
```

Comment:
```suggestion
  profileImageUrl?: string | null;
```
요렇게 옵셔널로 받고 아래에서 기본값을 null 로 주면 좋을것 같습니다!

Reply:
네 감사합니다!